gcloud-java-storage/src/test/java/com/google/gcloud/storage/SerializationTest.java

```diff
@@ -77,37 +73,21 @@
       Storage.BucketTargetOption.metagenerationNotMatch();
   private static final Map<StorageRpc.Option, ?> EMPTY_RPC_OPTIONS = ImmutableMap.of();
 
-  @Test
-  public void testServiceOptions() throws Exception {
+  @Override
+  public Serializable[] serializableObjects() {
     StorageOptions options = StorageOptions.builder()
         .projectId("p1")
         .authCredentials(AuthCredentials.createForAppEngine())
         .build();
-    StorageOptions serializedCopy = serializeAndDeserialize(options);
-    assertEquals(options, serializedCopy);
-
-    options = options.toBuilder()
+    StorageOptions otherOptions = options.toBuilder()
         .projectId("p2")
         .retryParams(RetryParams.defaultInstance())
```

Comment:
Unrelated to this PR: the `.retryParams(RetryParams.defaultInstance())` lines (present in a couple of the modules' serialization tests) doesn't test anything new anymore (compared to the `<Service>Options.defaultInstance()` options) because we switched retry params to be the default instance if none are specified (in ServiceOptions).  We can probably remove or switch those retry params to something else.